### ISSUE TITLE
Mixed 1-4 scaling?

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/benchmarksets#62]
@nhenriksen - we were trying to work with the cyclopentanol (guest 4) example from CD set 1, and noticed that the AMBER prmtop file has mixed 1-4 scaling factors (SCEE). Was this intended and, if so, can you explain why? 

This makes conversion into some code bases (GROMACS for example) impossible, and I've also never seen it before, so I am very curious where this came from/why it's done here.

Thanks.

cc @elkhoury

**Comment by nhenriksen** [comment]
We noticed this recently as well. I'm still trying to determine how best to proceed.  The reason it's there is that in the old days of AMBER, you could not specify the scaling factors in the parameter file (parm.dat or frcmod) when you specified dihedral parameters.  The cyclodextrin force field I am using (Q4MD-CD) was released right at the cusp of when support for specifying SCEE in the parameter file was added, via explicit "SCEE=1.2" type entries on the parameter line.  Due to a bug in tLEaP, the revised code inadvertently reads any trailing comments strings at the end of the parameter line, and, if the "SCEE=" entries are not present and the comment starts with a numeral, it would set SCEE to that numeric value. Until the code gets fixed, it is best to always explicitly provide the SCEE entry.

As far as I can tell from correspondence with Francois Dupradeau, who made the CD force field, they did not develop the force field with the newer AMBER version and so the messed up SCEE values were not present.  But for current users, the [downloadable force field files](http://upjv.q4md-forcefieldtools.org/REDDB/projects/F-85/) still do not contain explicit SCEE entries and so if they use them in AMBER, they will get the same results I did.

I've rerun some of the calculations with the correct SCEE values (by explicitly adding them to the parameter files) and I don't see statistically significant differences in the free energies and maybe just a hint of a very small difference in the binding enthalpies but I don't have enough data to be sure. The torsional histograms of the unbound CD host are very similar, so it's not causing a huge difference in the overall conformational properties of the CD.

So bottom line ... 1) I don't think it will influence the results.  2) I am planning to finish a complete set of calculations to check the impact. 3) I can update the benchmarks prmtop-rst7 files when this is complete and make a note about the force field parameter files.

**Comment by nhenriksen** [comment]
Side note, AMBER does support mixed 1-4 scaling factors because some of the carbohydrate force fields (Glycam) specifically do not use them (ie, set to 1.0).  The justification was a little unclear to me, but it was along the lines that they couldn't get anomeric effects quite right unless they set scaling to 1.0.  So If you need to do carbohydrate modified proteins in AMBER, it supports mixed scaling.
